common/djangoapps/student/views.py

```diff
@@ -2220,14 +2225,16 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
         )
 
     if request.method == 'POST':
-        password = request.POST['new_password1']
-        is_password_valid, password_err_msg = validate_password(user, password)
+        password1 = request.POST['new_password1']
+        password2 = request.POST['new_password2']
```

Comment:
Thanks @nasthagiri So it seems it's the matter of fact that we need to display form.errors on our template. I think it makes sense. 

I will update my PR accordingly.